plugins/available/alias-completion.plugin.bash

```diff
@@ -52,7 +52,7 @@ function alias_completion {
 		read -a alias_arg_words <<< "$alias_args"
 
 		# skip alias if there is no completion function triggered by the aliased command
-		if [[ ! " ${completions[*]} " =~ $alias_cmd ]]; then
+		if [[ ! $(_bash-it-array-contains-element "$alias_cmd" "${completions[*]}") ]]; then
```

Comment:
Great work discovering the existing function - I wonder how we can make things like this easier to know about...

If you look for uses of the function, namely the [test for it](https://github.com/Bash-it/bash-it/blob/e06066c36768ac08a31927646a150a6e66badbfc/test/lib/utilities.bats#L65), you'll see it uses `@` instead of `*`:
```
@test "_bash-it-array-contains-element() - when match is found, and is the first" {
  declare -a fruits=(apple pear orange mandarin)
  run has_match apple "${fruits[@]}"
  assert_line -n 0 'has apple'
}
```

I believe the `@` expands the array as individual elements vs a single string containing the elements.

Reply:
I see, fixed. Still works as expected :smile: